src/mmw/apps/home/views.py

```diff
@@ -13,10 +15,24 @@
 from apps.modeling.models import Project
 
 
+def get_stream_layers():
+    tiler_prefix = '//'
+    tiler_host = settings.TILER_HOST
+    tiler_postfix = '/{z}/{x}/{y}'
+    tiler_base = '%s%s' % (tiler_prefix, tiler_host)
+
+    stream_layers = deepcopy(settings.STREAM_LAYERS)
+    for layer in stream_layers:
+        layer['endpoint'] = urljoin(tiler_base, layer['code'] + tiler_postfix)
+
+    return stream_layers
```

Comment:
Avoid exposing the table name. See `api/modeling/boundary-layers/` for the data used to populate the other tiler data.